data/Gaussian/basicGaussian16/water_hf_solvent_ipcm.com

```diff
@@ -0,0 +1,13 @@
+%Chk="water.chk"
+%Rwf="water.rwf"
```

Comment:
Having this (and the checkpoint file) explicitly specified might cause problems when we run all of the calculations together when generating new test data.